scripts/install.sh

```diff
@@ -20,40 +20,42 @@ help()
    echo -e "\nUsage: ENV_VAR(S) $0"\
            "--config-dir <path>"\
            "--network <network>"\
-           "--contracts-only"
+           "--reuse-contracts"
    echo -e "\nEnvironment variables:\n"
    echo -e "\tKEEP_ETHEREUM_PASSWORD: The password to unlock local Ethereum accounts to set up delegations."\
            "Required only for 'local' network. Default value is 'password'"
    echo -e "\tCONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY: Contracts owner private key on Ethereum. Required for non-local network only"
    echo -e "\nCommand line arguments:\n"
    echo -e "\t--config-dir: Path to keep-core client configuration file(s)"
    echo -e "\t--network: Ethereum network for keep-core client."\
-                        "Available networks and settings are specified in the 'truffle-config.js'"
-   echo -e "\t--contracts-only: Should execute contracts part only."\
-                        "Client installation will not be executed.\n"
+                        "Available networks and settings are specified in the 'hardhat.config.ts'"
+   echo -e "\t--reuse-contracts: The already deployed contracts can be reused to speed up the development. Default is false.\n"\
+   "Client installation will not be executed.\n"
    exit 1 # Exit script after printing help
 }
 
 # Transform long options to short ones
 for arg in "$@"; do
   shift
   case "$arg" in
-    "--config-dir")     set -- "$@" "-c" ;;
-    "--network")        set -- "$@" "-n" ;;
-    "--contracts-only") set -- "$@" "-m" ;;
-    "--help")           set -- "$@" "-h" ;;
-    *)                  set -- "$@" "$arg"
+    "--config-dir")      set -- "$@" "-c" ;;
+    "--network")         set -- "$@" "-n" ;;
+    "--reuse-contracts") set -- "$@" "-d" ;;
+    "--help")            set -- "$@" "-h" ;;
+    *)                   set -- "$@" "$arg"
   esac
 done
 
+
+
 # Parse short options
 OPTIND=1
-while getopts "c:n:mh" opt
+while getopts "c:n:dh" opt
 do
    case "$opt" in
       c ) config_dir_path="$OPTARG" ;;
       n ) network="$OPTARG" ;;
-      m ) contracts_only=true ;;
+      d ) reuse_contracts=true ;;
```

Comment:
Should this be assigned to `true`, shouldn't it be `$OPTARG`?

Reply:
It worked with the original version as well, but I changed it to `$OPTARG`. Part of: [2dcb842](https://github.com/keep-network/keep-core/pull/3026/commits/2dcb842ce661077ccb2491e425aa08f4112bbdbe)